content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,375 @@
+---
+title: Pod Eviction
+content_type: concept
+weight: 50
+---
+
+{{< glossary_definition term_id="eviction" length="all" >}}
+
+During an eviction, the kubelet sets the `PodPhase` for the selected pods to `Failed`.
+This gracefully terminates the pods. 
```

Comment:
Note that it does not respect the pod's terminationGracePeriodSeconds, and uses a 0s grace-period.